src/components/FrameCustomiser.svelte

```diff
@@ -40,6 +40,14 @@
                 return `discord-icon=${this.value}`;
             },
         },
+        {
+            key: "flags",
+            name: "Show Flags",
```

Comment:
can we rename this from flags to badges? Seems more intuitive
```suggestion
            name: "Show Badges",
```